advanced-usage/index.md

```diff
@@ -179,9 +226,9 @@ There are buttons to
 In future it will be possible to set the current vehicle position, and click to guide the vehicle to new
 positions.
 
-##### Video
+##### Video Player
 
-The video widget displays an available WebRTC video stream. BlueOS uses the 
+The video player widget displays an available WebRTC video stream. BlueOS uses the 
```

Comment:
We should also add here that one can mirror the video in both horizontal and vertical axis, but that this configuration is not transmitted to the video recording, so if you have a camera that is upside down, you can use those controls to fix your view, but the video recording will be upside down nonethless.

We should also mention that one can (and should) select the desired video source IP in this configuration dialog, so the video is fixed in this source route and video stuttering is prevented.

Reply:
I need to find the PR for that change, and update the configuration image